scripts/functions

```diff
@@ -1305,6 +1305,24 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported) or using
+# fallback architecture-specific functions.
```

Comment:
As far as I understand it, if we have multilib support, we now do have
multi-arch support too (because of "Enable multiarch support in pass-1
GCC"), no?

So, gcc will alwaysupport -print-multiarch. Unless it may return an
empty set, but still have multilib variants?